testing/cluster_test_utils.py

```diff
@@ -37,6 +37,41 @@
 
 ten_minutes = 600  # number of seconds in 10 minutes
 
+cluster_cidr = "10.0.1.0/24"
+cluster_ip_prefix = "10.0.1"
+cluster_notation = 24
```

Comment:
There are redundant values here.  The latter two can be derived from the first.  Also, we know that these values can be problematic.  Why not start with 169.254.3.0/24 ?

Reply:
It's redundant, but in case set_cluster_cidr() doesn't get called, I'd like for cluster_ip_prefix and cluster_notation to have reasonable defaults.

I can start with 169.254.3.0/24